hibernate-core/src/test/java/org/hibernate/test/legacy/FooBarTest.java

```diff
@@ -615,7 +616,7 @@ public void testQuery() throws Exception {
 		s.save(bar);
 		s.save(baz);
 		list = s.createQuery(
-				" from Bar bar where bar.baz.count=667 and bar.baz.count!=123 and not bar.baz.name='1-E-1'"
+				" from Bar bar where bar.baz.count=667 and bar.baz.count<>123 and not bar.baz.name='1-E-1'"
```

Comment:
This one might be an issue.  If I remember correctly, "<>" isn't supported by several DBs.  May need to revert this fix and just skip the test.